tests/vite-app/vite.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from "vite";
+import alias from "@rollup/plugin-alias";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    rollupOptions: {
+      plugins: [
+        alias({
+          entries: [
+            {
+              find: "#node-web-compat",
```

Comment:
Let's add a comment why this is needed

Reply:
we switched to subpaths because of dist size - do we still like that decision if we require config like this?

also, will probably need to add a detailed README somewhere relating to using in a web apps and which bundlers work as expected, and which need tweaks like this